src/components/app/data/services/enterpriseCustomerUser.ts

```diff
@@ -145,16 +143,16 @@ export async function fetchEnterpriseLearnerData(username, enterpriseSlug, optio
 export async function fetchEnterpriseCourseEnrollments(enterpriseId, options = {}) {
   const queryParams = new URLSearchParams({
     enterprise_id: enterpriseId,
-    is_active: true,
+    is_active: 'true',
     ...options,
   });
   const url = `${getConfig().LMS_BASE_URL}/enterprise_learner_portal/api/v1/enterprise_course_enrollments/?${queryParams.toString()}`;
   try {
     const response = await getAuthenticatedHttpClient().get(url);
     return camelCaseObject(response.data);
   } catch (error) {
-    if (getErrorResponseStatusCode(error) !== 404) {
-      logError(error);
+    if (getErrorResponseStatusCode(error as Error) !== 404) {
+      logError(error as Error);
     }
```

Comment:
nit: to avoid multiple `as Error`, you might also consider:

```ts
} catch (error) {
  if (error instanceof Error && getErrorResponseStatusCode(error) !== 404)) {
    logError(error);
  }
  return [];
}
```